### PR TITLE
21676 - Add in credits for is_reversal also add in refund_date

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:*
https://github.com/bcgov/entity/issues/21676

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

**Comment by seeker25** [comment]
Need to update the job hash..

File: jobs/payment-jobs/pyproject.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.12"
-pay-api = {git = "https://github.com/bcgov/sbc-pay.git", subdirectory = "pay-api"}
+pay-api = {git = "https://github.com/seeker25/sbc-pay.git", branch = "disbursement_refund_date_fix", subdirectory = "pay-api"}
```
Comment (seeker25) [inline jobs/payment-jobs/pyproject.toml:10]: I will adjust these when we push to PROD

**Comment by seeker25** [comment]
I did the same work in:
https://github.com/bcgov/sbc-pay/compare/main...release/1.20.9

**Comment by seeker25** [comment]
Will make a unit test.. one sec

**Review by avni-work — APPROVED** [review]

**Pull request merged**